sklearn/multiclass.py

```diff
@@ -409,6 +476,46 @@ def fit(self, X, y):
 
         return self
 
+    def partial_fit(self, X, y, classes=None):
+        """Partially fit underlying estimators 
+        
+        Should be used when memory is inefficient to train all data. Chunks 
+        of data can be passed in several iteration, where the first call 
+        should have an array of all target variables.
+
+
+        Parameters
+        ----------
+        X : (sparse) array-like, shape = [n_samples, n_features]
+            Data.
+
+        y : array-like, shape = [n_samples]
+            Multi-class targets.
+
+        Returns
+        -------
+        self
+        """
+        if self._first_partial_run:
+            if classes is None:
+                raise ValueError("Must contain all classes of target")
```

Comment:
Same remark about dostring of the function and error message. This error message is a bit hard to understand outside of the context.